Gemfile

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
```

Comment:
could you at least delete/change a name of "Uploaded first version of regression for Auction
"